source/YugiAcademy/YuGiOh/migrations/0002_rename_card_type_boosterpackcard_content_type_and_more.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.1 on 2022-09-18 04:26
```

Comment:
This file need to be pushed?

Reply:
Yes, because I had to renamed some instance variables from Booster Pack Card and those instance variables map each one to a column.